person.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'nameable'
+
+class Person < Nameable
+  attr_accessor :name, :age
+  attr_reader :id, :rentals
+
+  def initialize(name: 'Unknown', age: 0, parent_permission: true)
+    super()
+    @id = rand(1000..9999)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @rentals = []
+  end
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+
+  def correct_name
+    @name
+  end
+
+  def rent_books(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] This method is for adding `rentals` and remembering `the person` itself should be sent as a parameter to create the new rental achieving the 'has-many' association also we need a `book` you want to rent and `date`
```ruby
def rent_books(book, date)
    Rental.new(date, book, self)
  end
```